frontend/talentsearch/src/js/components/search/SearchContainer.tsx

```diff
@@ -180,18 +185,18 @@ export const SearchContainer: React.FC<SearchContainerProps> = ({
           >
             {intl.formatMessage(
               {
-                defaultMessage: `{candidateCount, plural,
-                  =0 {Results: <testId>{candidateCount}</testId> matching candidates}
-                  =1 {Results: <testId>{candidateCount}</testId> matching candidate}
-                  other {Results: <testId>{candidateCount}</testId> matching candidates}
+                defaultMessage: `{totalCandidateCount, plural,
+                  =0 {Results: <testId>{totalCandidateCount}</testId> matching candidates}
+                  =1 {Results: <testId>{totalCandidateCount}</testId> matching candidate}
+                  other {Results: <testId>{totalCandidateCount}</testId> matching candidates}
                 }`,
-                id: "aWLo7o",
+                id: "eeWkWi",
```

Comment:
please update the Fr.json file for this disconnected Intl

Reply:
:ninja: !

Reply:
Fixed by [a760102](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4528/commits/a760102652f60307ef6993c676cdf99838eef0bc)